.github/workflows/azure-snp-reporter.yml

```diff
@@ -20,6 +20,7 @@ jobs:
         uses: actions/setup-go@4d34df0c2316fe8122ab82dc22947d607c0c91f9 # v4.0.0
         with:
           go-version: 1.20.4
+          cache: false
```

Comment:
Does this job in the workflow actually require Go?
There might be a dependency on Go very deep down, but just from going over it quickly it doesnt seem like we are executing any Go commands